lib/elixir/src/elixir_module.erl

```diff
@@ -229,6 +219,28 @@ build(Line, File, Module, Lexical) ->
 
   {Data, Defs, Ref}.
 
+compile_definition_attributes(#{module := Module} = Env, Kind, Name, Args, _Guards, _Body) ->
+  Data = data_table(Module),
+  Arity = length(Args),
+  Tuple = {Name, Arity},
+
+  %% Docs must come first as they read the impl callback
+  case elixir_compiler:get_opt(docs) of
```

Comment:
Let's skip this check. We already skip adding the chunk later in `elixir_erl`, this way we can hide the compilation of all attributes behind the bootstrap flag.